app/components/Header/DropdownContents/InfoDropdown/index.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import Icon from 'app/components/Icon';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+import styles from './InfoDropdown.css';
+import shared from 'app/components/Header/Header.css';
+
+const InfoDropdown = () => (
+  <div className={styles.infoDropdownEl}>
+    <div
+      className={cx(shared.dropdownSection, styles.gridSection)}
+      data-first-dropdown-section
+    >
+      <Link to="pages/info-om-abakus">
+        <div>
+          <span>Generelt</span>
+          <Icon className={styles.icon} name="arrow-forward" size={20} />
+        </div>
+        <p>Litt av hvert om Abakus</p>
+      </Link>
```

Comment:
Please dear lord separate this into it's own component...

Reply:
haha I considered it, and then forgot ... 😅 Will do!